src/framework/components/scroll-view/component.js

```diff
@@ -32,6 +33,8 @@ var _tempScrollValue = new Vec2();
  *
  * @property {number} bounceAmount Controls how far the content should move before bouncing back.
  * @property {number} friction Controls how freely the content should move if thrown, i.e. By flicking on a phone or by flinging the scroll wheel on a mouse. A value of 1 means that content will stop immediately; 0 means that content will continue moving forever (or until the bounds of the content are reached, depending on the scrollMode).
+ * @property {boolean} useMouseWheel Whether to use mouse wheel for scrolling (horizontally and vertically).
+ * @property {Vec2} mouseWheelSensitivity Mouse wheel horizontal and vertical sensitivity. Only used if useMouseWheel is set.
```

Comment:
What is the range here?

Reply:
No particular range here. Developers can set to 0 if they want to disable one of the directions, [0..1] if they want slower than default, or [1..Inf] if they want faster. For UI-heavy games (strategy, 4X, etc), this would most likely be an in-game setting.

Reply:
Might be worth adding some of this info. e.g. "Setting a direction to 0 will disable mouse wheel scrolling in that direction. 1 is a default sensitivity that is considered to feel good. The values can be set higher or lower than 1 to tune the sensitivity. Set to [1, 1] by default.".

Reply:
Nice, I like it!!